cdap-master/pom.xml

```diff
@@ -93,6 +93,11 @@
       <artifactId>cdap-app-fabric</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-metrics-sidecar</artifactId>
```

Comment:
Please don't call a maven module `sidecar`. Sidecar or not is an implementation details.

Reply:
Renamed the module to cda-system-metrics